csm_web/scheduler/factories.py

```diff
@@ -275,7 +275,7 @@ def generate_test_data(complicate=False):
     if not settings.DEBUG:
         print("This cannot be run in production! Aborting.")
         return
-    management.call_command("flush", interactive=True)
+    management.call_command("flush", interactive=False)
```

Comment:
I think it's still worth having interactive=True, because even though the if statement above stops you from running this in production, this check has stopped me from accidentally wiping the database on my test environment a couple times.